insights_client/__init__.py

```diff
@@ -30,13 +31,16 @@
 try:
     insights_uid = pwd.getpwnam("insights").pw_uid
     insights_gid = pwd.getpwnam("insights").pw_gid
+    insights_grpid = grp.getgrpname("insights").gr_gid
+    insights_grpusers = grp.getgrpname("insights").gr_mem
+    curr_user_grps = os.getgroups()
 except:
-    insights_uid, insights_gid = None, None
-    raise
+    sys.exit("User and group 'insights' not found. Exiting.")
 
 
 def demote(uid, gid, phase):
-    if uid and gid and phase != "collect":
+    user_is_root = os.geteuid() is 0
+    if not user_is_root:
```

Comment:
Small nitpick - IMHO `if os.geteuid() == 0` is just as readable.

Reply:
Do you mean `if os.geteuid() != 0`?